hyperon_das_atomdb/exceptions.py

```diff
@@ -44,3 +44,7 @@ class RetryException(BaseException):
 
 class InvalidAtomDB(BaseException):
     ...  # pragma no cover
+
+
+class ParserException(BaseException):
```

Comment:
This name is too generic. Firstly. I suggest InvalidSQL. We're not claiming to actually parse SQL. To do that we would need to be prepared for anything that's possible in SQL and this definitely is not the case. We want to be able to not implement some specific SQL feature which is particularly difficult to map to our Atomspace.

Reply:
Done!